README.md

```diff
@@ -19,3 +19,65 @@ files from [server](./templates/repository/server) and libraries from the
 
 To update the repositories simply make your changes. Once merged to master, they
 will be published using a GitHub Action.
+
+## Github Sync action
+
+The scripts in "/scripts" serve to synchronize all ORY repositories to a common
+template, including README, CONTRIBUTING, COC, LICENCE, Github Workflows with
+close to zero manual interaction. Depending on repository type (server, library,
+action) specific templates can be copied as well. The project names, links to
+documentation ect. are being substituted for each project in "/scripts/sync.sh".
+For more details please refer to the documentation within the scripts.
+
+#### Documentation of sync.yml
+
+```yml
+name: Synchronize Repositories
```

Comment:
You could probably document this in the sync.yml file itself and link to it here?

Reply:
👍 
I was not sure if the comments would be too much in the sync.yml, but if they dont bother that would look smoother!